glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -21,6 +21,11 @@ import mozilla.telemetry.glean.internal.InternalConfiguration
 import mozilla.telemetry.glean.internal.initialize as gleanInitialize
 import mozilla.telemetry.glean.internal.enableLogging
 import mozilla.telemetry.glean.internal.finishInitialize
+import mozilla.telemetry.glean.internal.setExperimentActive as gleanSetExperimentActive
+import mozilla.telemetry.glean.internal.setExperimentInactive as gleanSetExperimentInactive
+import mozilla.telemetry.glean.internal.testIsExperimentActive as gleanTestIsExperimentActive
+import mozilla.telemetry.glean.internal.testGetExperimentData as gleanTestGetExperimentData
+import mozilla.telemetry.glean.internal.*
```

Comment:
I am considering just naming the top-level functions with a `glean_` prefix in the UDL. 
That avoids the renaming here in Kotlin and the wrapping we will need to do in Swift.

@travis, wdyt?